Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -8273,6 +8296,34 @@ def set_feed_last_run(last_run_indicators):
         demisto.setIntegrationContext(last_run_indicators)
 
 
+def set_last_mirror_run(last_mirror_run):
+    """
+    This function sets the mirror last run: from XSOAR version 6.6.0: using `demisto.setLastMirrorRun()`.
+    Before XSOAR version 6.6.0: we don't set the given data`.
+    :type last_mirror_run: ``dict``
+    :param last_mirror_run: Data to save in "LastMirrorRun" object.
+    :rtype: ``None``
+    :return: None
+    """
+    if is_demisto_version_ge('6.6.0'):
+        demisto.setLastMirrorRun(last_mirror_run)
+    else:
+        return "You cannot use setLastMirrorRun as your version is below 6.6.0"
```

Comment:
I am not sure about the else. @anara123 what do you say?

Reply:
Keep it. This is ok this way

Reply:
raise DemistoException instead, don't return string.